browser-extension/mv2/src/background/background.js

```diff
@@ -264,13 +269,50 @@ BG.Methods.modifyHeaders = function (originalHeaders, headersTarget, details) {
     ruleType,
     rulePairs,
     rulePair,
-    isRuleApplied = false,
```

Comment:
`isRuleApplied` is a convention used across all the rule type execution logic So we should stick to it.